### ISSUE TITLE
tracking.es6:63-64: Create tests for eCommerce tracking....

### ISSUE THREAD

**Issue by 0pdd** [fidals/shopelectro#762]
The puzzle `759-e8d84e6c` from #759 has to be resolved: 

https://github.com/fidals/shopelectro/blob/987327e514575142670d965adc25bc3203bf9f8d/front/js/shared/tracking.es6#L63-L64

The puzzle was created by Artemiy on 07-Mar-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by artemiy312** [comment]
I increased the priority, because it transitively blocks fixes of google eCommerce for STB

**Comment by artemiy312** [comment]
first i'll create tests for google to unblock  #761

**Comment by artemiy312** [comment]
I want to use `shopelectro.selenium` classes for new tests, so i will do this issue with  #764

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772) are still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#785](https://github.com/fidals/shopelectro/issues/785) is still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#799](https://github.com/fidals/shopelectro/issues/799) is still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#808](https://github.com/fidals/shopelectro/issues/808) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#920](https://github.com/fidals/shopelectro/issues/920), [#921](https://github.com/fidals/shopelectro/issues/921) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799), [#808](https://github.com/fidals/shopelectro/issues/808).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#920](https://github.com/fidals/shopelectro/issues/920), [#921](https://github.com/fidals/shopelectro/issues/921), [#929](https://github.com/fidals/shopelectro/issues/929), [#930](https://github.com/fidals/shopelectro/issues/930) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799), [#808](https://github.com/fidals/shopelectro/issues/808).

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#929](https://github.com/fidals/shopelectro/issues/929), [#930](https://github.com/fidals/shopelectro/issues/930), [#938](https://github.com/fidals/shopelectro/issues/938) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799), [#808](https://github.com/fidals/shopelectro/issues/808), [#920](https://github.com/fidals/shopelectro/issues/920), [#921](https://github.com/fidals/shopelectro/issues/921).

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#930](https://github.com/fidals/shopelectro/issues/930), [#938](https://github.com/fidals/shopelectro/issues/938), [#940](https://github.com/fidals/shopelectro/issues/940) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799), [#808](https://github.com/fidals/shopelectro/issues/808), [#920](https://github.com/fidals/shopelectro/issues/920), [#921](https://github.com/fidals/shopelectro/issues/921), [#929](https://github.com/fidals/shopelectro/issues/929).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#807](https://github.com/fidals/shopelectro/issues/807), [#930](https://github.com/fidals/shopelectro/issues/930), [#940](https://github.com/fidals/shopelectro/issues/940) are still not solved; solved: [#771](https://github.com/fidals/shopelectro/issues/771), [#772](https://github.com/fidals/shopelectro/issues/772), [#785](https://github.com/fidals/shopelectro/issues/785), [#799](https://github.com/fidals/shopelectro/issues/799), [#808](https://github.com/fidals/shopelectro/issues/808), [#920](https://github.com/fidals/shopelectro/issues/920), [#921](https://github.com/fidals/shopelectro/issues/921), [#929](https://github.com/fidals/shopelectro/issues/929), [#938](https://github.com/fidals/shopelectro/issues/938).
